account_fiscal_position_vat_check/__openerp__.py

```diff
@@ -23,7 +23,7 @@
 
 {
     'name': 'Account Fiscal Position VAT Check',
-    'version': '0.1',
+    'version': '8.0.0.1.1',
```

Comment:
I prefer not as there is nothing new just a bug fix. Current version being implicitly `8.0.0.1`